client/tracks/index.js

```diff
@@ -60,6 +60,10 @@ const events = {
 	CONNECT_ACCOUNT_LEARN_MORE: 'wcpay_welcome_learn_more',
 	CONNECT_ACCOUNT_STRIPE_CONNECTED: 'wcpay_stripe_connected',
 	CONNECT_ACCOUNT_KYC_MODAL_OPENED: 'wcpay_connect_account_kyc_modal_opened',
+	OVERVIEW_DEPOSITS_VIEW_HISTORY_CLICK:
+		'wcpay_overview_deposits_view_history_click',
+	OVERVIEW_DEPOSITS_CHANGE_SCHEDULE_CLICK:
+		'wcpay_overview_deposits_change_schedule_click',
```

Comment:
I've only added the two most important interactions. See [issue](https://github.com/Automattic/woocommerce-payments/issues/5881#issuecomment-1530793008) for discussion of other possible events to track. 

Happy to add more events here or as follow up PRs.